### ISSUE TITLE
R package migration to conda-forge

### ISSUE THREAD

**Issue by daler** [bioconda/bioconda-recipes#5582]
In preparation to supporting R 3.4, we are migrating any non-bio-related packages over to the conda-forge channel. This issue aims to organize and track the progress.

Below are the results of:

```bash
bioconda-utils dag recipes/ config.yml --format txt --packages "r-*" \
   | sed "/recipes/ s|recipes/|- [ ] |"
```

# Workflow
The list is divided into independent subdags. Each subdag is topologically sorted such that packages at the top are dependencies of packages at the bottom. Within each subdag, recipes should be built in order.

- Working from the top, inspect packages to see:
  -  if a recent version already exists in conda-forge, indicate the version number and tick the checkbox for the package in this list (if there's more than one version here, tick them all)
  - if they are bio-related, in which case add `**bio-related**` and a short description. These should not be submitted to conda-forge, but we want to keep track of them. We will come through in another pass to work on these. 
- Submit chunks of 10 or so off the top and submit to conda-forge. Each recipe submitted to conda-forge should be the most recent version available of the package. Bjoern's  https://github.com/bgruening/conda_r_skeleton_helper helps a lot with this.
- After submitting a PR to conda-forge, edit this post and add the PR next to it.
- Once the PR is merged into conda-forge, tick the checkbox.

# subdag 0
- [x] r-svgui https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-catools https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-misc3d https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-svdialogs https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-proj4 https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-extrafontdb https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-dendextend
- [x] r-pracma https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-r.huge https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-pcapp https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-aroma.apd https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-rrcov https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-purrr https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-png https://github.com/conda-forge/staged-recipes/pull/3629
- [x] ~~r-htmltools~~ (0.3.6 already in conda-forge)
- [x] r-hwriter https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-flashclust https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-ggdendro https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-fastcluster https://github.com/conda-forge/staged-recipes/pull/3629
- [x] r-locfit https://github.com/conda-forge/staged-recipes/pull/3629
  - [x] r-corpcor https://github.com/conda-forge/staged-recipes/pull/3629 -bag
  - [x] r-rttf2pt1 https://github.com/conda-forge/staged-recipes/pull/3629 -bag
- [x] r-extrafont https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-ellipse https://github.com/conda-forge/staged-recipes/pull/3649
- [x] ~~r-mixomics~~  **bio-related** (multi-omics integration)
- [x] ~~r-data.table/1.10.4~~ (already in conda-forge)
- [x] ~~r-solarius~~ **bio-related** (linkage and association of QTLs)
- [x] ~~r-wasabi~~ **bio-related** (prepares Sailfish/Salmon output)
- [x] r-r.cache https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-r.rsp https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-permute https://github.com/conda-forge/staged-recipes/pull/3649
- [x] ~~r-mutationalpatterns~~  **bio-related** (visualize base substitutions)
- [x] ~~r-mitools~~ **bio-related?** (multiple-imputation datasets; examples are bio-related but not sure how general this is)
- [x] r-ggrepel https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-ptw https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-gplots https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-gmd https://github.com/conda-forge/staged-recipes/pull/3649
- [x] ~~r-pore~~ **bio-related** (nanopore sequencing)
- [x] ~~r-dynamictreecut~~ **bio-related?** (dynamic tree cutting; used in coexpression network building)
- [x] r-longitudinal https://github.com/conda-forge/staged-recipes/pull/3649
- [x] ~~r-rubic/1.0.2~~ **bio-related** (copy number aberrations)
- [x] ~~r-rubic/1.0.3~~ **bio-related** (copy number aberrations)
- [x] ~~r-globals/0.6.0~~ (not migrating old version)
- [x] r-globals https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-scatterplot3d https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-leaps https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-splitstackshape https://github.com/conda-forge/staged-recipes/pull/3649
- [x] ~~r-spp~~ **bio-related** (ChIP-seq peak-calling)
- [x] ~~r-vegan/2.3_4~~ (2.4.3 in conda-forge)
- [x] ~~r-vegan/2.3_0~~ (2.4.3 in conda-forge)
- [x] ~~r-vegan/2.3_3~~ (2.4.3 in conda-forge)
- [x] r-multicool https://github.com/conda-forge/staged-recipes/pull/3649
- [x] r-listenv https://github.com/conda-forge/staged-recipes/pull/3827
- [x] ~~r-listenv/0.5.0~~ (skipping older version)
- [x] ~~r-fnn~~ (1.1 in conda-forge)
- [x] r-ks  https://github.com/conda-forge/staged-recipes/pull/3827
- [x] ~~r-ks/1.10.5~~ (skipping old version)
- [x] r-ash  https://github.com/conda-forge/staged-recipes/pull/3827
- [x] r-hdrcde  https://github.com/conda-forge/staged-recipes/pull/3827
- [x] r-fdrtool
- [x] ~~r-genenet~~ *bio-related** (gene networks)
- [x] ~~r-matrixstats~~ (0.52.2 in conda-forge)
- [x] ~~r-samr~~ **bio-related** (microarrays)
- [x] ~~r-soap-nmr~~ **bio-related** (NMR data)
- [x] ~~r-future/0.9.0~~ (skipping old version)
- [x] r-future https://github.com/conda-forge/staged-recipes/pull/3827
- [x] r-r.filesets  https://github.com/conda-forge/staged-recipes/pull/3827
- [x] ~~r-pscbs~~ **bio-related** (allele-specific CNV detection)
- [x] ~~r-pscbs/0.61.0~~ (skipping older version)
- [x] r-survey  https://github.com/conda-forge/staged-recipes/pull/3827
- [x] r-relaimpo  https://github.com/conda-forge/staged-recipes/pull/3827
- [x] r-bma  https://github.com/conda-forge/staged-recipes/pull/3827
- [x] ~~r-coloc~~ **bio-related** (SNP colocalization)
- [x] ~~r-wgcna~~ **bio-related** (gene network clustering)
- [x] ~~r-yaml~~ (2.1.14 in conda-forge)
- [x] ~~r-knitr~~ (1.16 in conda-forge)
- [x] r-factominer
- [x] ~~r-rmarkdown~~ (1.6 in conda-forge)
- [x] r-knitrbootstrap https://github.com/conda-forge/staged-recipes/pull/3977
- [x] ~~r-sartools~~ **bio-related** (RNA-seq specific tools; bioconductor dependencies)
- [x] r-rainbow
- [x] ~~r-mmgenome~~ **bio-related** (extracting genomes from metagenomes)
- [x] ~~r-sleuth~~ **bio-related** (differential expression from RNA-seq data)
- [x] ~~r-probmetab~~ **bio-related** (mass-spec related; bioconductor dependencies)
- [x] ~~r-ampvis~~ **bio-related** (visualization of amplicon data)
- [x] r-r.devices
- [x] ~~r-aroma.core~~ **bio-related** (microarrays)
- [x] ~~r-aroma.affymetrix~~ **bio-related** (microarrays)
- [x] ~~r-jsonlite~~ (1.4 in conda-forge)
- [x] ~~r-htmlwidgets~~ (0.8 in conda-forge)
- [x] r-d3heatmap
- [x] r-plotly
- [x] ~~r-plasmidprofiler~~ **bio-related** (working with plasmids)
- [x] r-ggalt
- [x] ~~r-rentrez~~ **bio-related** (interface to NCBI EUtils)

# subdag 1
- [x] r-spam https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-fields https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-mba https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-plotrix https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-splancs https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-gmodels https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-rpmg https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-geomap https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-algdesign https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-deldir https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-phonr https://github.com/conda-forge/staged-recipes/pull/3828
- [x] r-combinat https://github.com/conda-forge/staged-recipes/pull/3859
- [x] r-klar https://github.com/conda-forge/staged-recipes/pull/3859
- [x] r-learnbayes https://github.com/conda-forge/staged-recipes/pull/3859
- [x] r-spdep https://github.com/conda-forge/staged-recipes/pull/3859
- [x] ~~r-mutoss~~ **bio-related** (multiple hypothesis testing; bioconductor dependencies)
- [x] ~~r-drc~~ **bio-related** (dose-response analysis)
- [x] r-agricolae https://github.com/conda-forge/staged-recipes/pull/3859
- [x] r-rwave https://github.com/conda-forge/staged-recipes/pull/3859
- [x] r-rseis https://github.com/conda-forge/staged-recipes/pull/3859
- [x] r-rfoc https://github.com/conda-forge/staged-recipes/pull/3859

# subdag 2
- [x] r-shape https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-diagram https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-params https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-flowr https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-globaloptions https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-circlize https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-getoptlong https://github.com/conda-forge/staged-recipes/pull/3860

# subdag 3
- [x] ~~r-diptest/0.75_7~~ (0.75.7 in conda-forge)
- [x] r-mclust/5.2 https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-mclust https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-trimcluster/0.1_2 https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-flexmix/2.3.14 https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-kernlab/0.9_25 https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-prabclus/2.2_6 https://github.com/conda-forge/staged-recipes/pull/3860
- [x] r-fpc/2.1.10 https://github.com/conda-forge/staged-recipes/pull/3860

# subdag 4
- [x] r-ade4 https://github.com/conda-forge/staged-recipes/pull/3826
- [x] ~~r-seqinr~~ **bio-related** (DNA & protein seq analysis and viz)
- [x] ~~r-cnogpro~~ **bio-related** (copy number and breakpoints in prokaryotes)
- [x] ~~r-mlgt~~ **bio-related** (454 pyrosequencing analysis)

# subdag 5
- [x] r-bbmisc https://github.com/conda-forge/staged-recipes/pull/3826
- [x] r-fail https://github.com/conda-forge/staged-recipes/pull/3826
- [x] r-sendmailr https://github.com/conda-forge/staged-recipes/pull/3826
- [x] r-batchjobs https://github.com/conda-forge/staged-recipes/pull/3826

# subdag 6
- [x] r-biglm https://github.com/conda-forge/staged-recipes/pull/3826
- [x] r-bigmemory.sri https://github.com/conda-forge/staged-recipes/pull/3826
- [x] r-bigmemory https://github.com/conda-forge/staged-recipes/pull/3826
- [x] r-biganalytics https://github.com/conda-forge/staged-recipes/pull/3826

# subdag 7
- [x] r-futile.options https://github.com/conda-forge/staged-recipes/pull/3779
- [x] r-lambda.r https://github.com/conda-forge/staged-recipes/pull/3779
- [x] r-futile.logger https://github.com/conda-forge/staged-recipes/pull/3826
- [x] r-venndiagram https://github.com/conda-forge/staged-recipes/pull/3826

# subdag 8
- [x] r-prettyunits https://github.com/conda-forge/staged-recipes/pull/3790
- [x] r-progress https://github.com/conda-forge/staged-recipes/pull/3790
- [x] ~~r-reshape/0.8.6~~ (0.8.6 in conda-forge)
- [x] r-ggally/0.5.0 (omitting older version)
- [x] r-ggally https://github.com/conda-forge/staged-recipes/pull/3790

# subdag 9
- [x] ~~r-readmzxmldata~~ **bio-related** (reading mass spec data)
- [x] ~~r-readbrukerflexdata~~ **bio-related** (reading mass spec data)
- [x] ~~r-maldiquant~~ **bio-related** (mass spec analysis pipeline)
- [x] ~~r-maldiquantforeign~~ **bio-related** (support for maldiquant mass spec package)

# subdag 10
- [x] r-gamlss.data https://github.com/conda-forge/staged-recipes/pull/3790
- [x] r-gamlss.dist https://github.com/conda-forge/staged-recipes/pull/3790
- [x] r-gamlss https://github.com/conda-forge/staged-recipes/pull/3790

# subdag 11
- [x] r-googlevis https://github.com/conda-forge/staged-recipes/pull/3790
- [x] r-runit https://github.com/conda-forge/staged-recipes/pull/3790
- [x] ~~r-nanostringnorm~~ **bio-related** (working with NanoString data)

# subdag 12
- [x] ~~r-argparse~~ (1.0.4 in conda-forge)
- [x] ~~r-boutroslabplottinggeneral~~ **bio-related** (plotting specific kinds of biological data. Note that this is not on CRAN)

# subdag 13
- [x] r-aod https://github.com/conda-forge/staged-recipes/pull/3790
- [x] ~~r-exomedepth~~ **bio-related** (CNV calling from exome-seq)

# subdag 14
- [x] r-bbmle https://github.com/conda-forge/staged-recipes/pull/3790
- [x] r-emdbook https://github.com/conda-forge/staged-recipes/pull/3790

# subdag 15
- [x] r-optparse https://github.com/conda-forge/staged-recipes/pull/3881
- [x] ~~r-histonehmm~~ **bio-related** (histone mod ChIP-seq. Not a CRAN or Bioconductor package)

# subdag 16
- [x] r-itertools https://github.com/conda-forge/staged-recipes/pull/3851
- [x] r-missforest https://github.com/conda-forge/staged-recipes/pull/3851

# subdag 17
- [x] r-locfdr https://github.com/conda-forge/staged-recipes/pull/3851
- [x] r-nhmmfdr https://github.com/conda-forge/staged-recipes/pull/3851

# subdag 18
- [x] r-misctools https://github.com/conda-forge/staged-recipes/pull/3851
- [x] r-maxlik https://github.com/conda-forge/staged-recipes/pull/3851

# subdag 19
- [x] ~~r-microseq~~ **bio-related** (microbial sequence analysis)
- [x] ~~r-micropan~~ **bio-related** (visualize microbial genomes)

# subdag 20
- [x] r-segmented https://github.com/conda-forge/staged-recipes/pull/3851
- [x] r-mixtools https://github.com/conda-forge/staged-recipes/pull/3851

# subdag 21
- [x] r-ore https://github.com/conda-forge/staged-recipes/pull/3837
- [x] r-rniftyreg https://github.com/conda-forge/staged-recipes/pull/3837

# subdag 22
- [x] r-pheatmap https://github.com/conda-forge/staged-recipes/pull/3837
- [x] ~~r-qiimer~~ **bio-related** (work with QIIME output)

# subdag 23
- [x] r-polynom https://github.com/conda-forge/staged-recipes/pull/3837
- [x] r-preseqr https://github.com/conda-forge/staged-recipes/pull/3837

# subdag 24
- [x] r-sets https://github.com/conda-forge/staged-recipes/pull/3837
- [x] r-relations https://github.com/conda-forge/staged-recipes/pull/3837

# subdag 25
- [x] r-truncnorm https://github.com/conda-forge/staged-recipes/pull/3778
- [x] r-rsolnp https://github.com/conda-forge/staged-recipes/pull/3837

# subdag 26
- [x] ~~r-snow~~ (0.4.2 in conda-forge)
- [x] r-snowfall https://github.com/conda-forge/staged-recipes/pull/3779

# singletons
- [x] ~~r-abind~~ version 1.4_5 in conda-forge
- [x] r-ada https://github.com/conda-forge/staged-recipes/pull/3757 
- [x] r-ahocorasicktrie https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-amap https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-argparser/0.4 https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-base64 https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-batch https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-beanplot https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-biasedurn https://github.com/conda-forge/staged-recipes/pull/3757
- [x] ~~r-bio3d~~  **bio-related** (protein structure and MD data manipulation)
- [x] ~~r-biom~~ **bio-related** (BIOM format)
- [x] r-blockmodeling https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-calibrate https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-caroline https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-cghflasso https://github.com/conda-forge/staged-recipes/pull/3757
- [x] r-changepoint https://github.com/conda-forge/staged-recipes/pull/3757
- [x] ~~r-chbutils~~ **bio-related** (bioinf tools)
- [x] r-checkpoint https://github.com/conda-forge/staged-recipes/pull/3881
- [x] ~~r-cimpl~~ **bio-related** (mutagenesis analysis)
- [x] r-compquadform https://github.com/conda-forge/staged-recipes/pull/3881
- [x] r-compute.es https://github.com/conda-forge/staged-recipes/pull/3881
- [x] r-corrplot https://github.com/conda-forge/staged-recipes/pull/3881
- [x] ~~r-dbchip~~ **bio-related** (ChIP-seq differential binding. In bioconductor, not CRAN)
- [x] r-delaporte https://github.com/conda-forge/staged-recipes/pull/3881
- [x] ~~r-deoptimr~~ (1.0.8 in conda-forge)
- [x] r-discriminer https://github.com/conda-forge/staged-recipes/pull/3881
- [x] r-docopt https://github.com/conda-forge/staged-recipes/pull/3778
- [x] r-dorng https://github.com/conda-forge/staged-recipes/pull/3881
- [x] r-downloader https://github.com/conda-forge/staged-recipes/pull/3881
- [x] ~~r-dpeak~~ **bio-related** (ChIP-seq peak deconvolution. Neither CRAN nor Bioconductor package)
- [x] r-dunn.test https://github.com/conda-forge/staged-recipes/pull/3882
- [x] r-e1071/1.6_8 https://github.com/conda-forge/staged-recipes/pull/3882
- [x] ~~r-empiricalfdr.deseq2~~ **bio-related** (work with DESeq2 output; Bioconductor deps but not in bioconductor)
- [x] r-exactranktests https://github.com/conda-forge/staged-recipes/pull/3882
- [x] r-fastmatch https://github.com/conda-forge/staged-recipes/pull/3882
- [x] r-fda https://github.com/conda-forge/staged-recipes/pull/3882
- [x] r-ff https://github.com/conda-forge/staged-recipes/pull/3882
- [x] r-fftwtools https://github.com/conda-forge/staged-recipes/pull/3882
- [x] r-fitdistrplus https://github.com/conda-forge/staged-recipes/pull/3882
- [x] r-flexclust/1.3_4 https://github.com/conda-forge/staged-recipes/pull/3882
- [x] r-funr https://github.com/conda-forge/staged-recipes/pull/3882
- [x] r-gam (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-gbm (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-gdtools (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-genabel.data (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-ggthemes (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-giant (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-gprofiler (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-gpseq (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-grimport (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-gsalib (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-gsmoothr (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-idpmisc (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-idr (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-inlinedocs (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-km.ci (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-kmsurv (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-kriging (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-lsd (https://github.com/conda-forge/staged-recipes/pull/3785)
- [x] r-matlab https://github.com/conda-forge/staged-recipes/pull/3883
- [x] ~~r-matrixeqtl~~ **bio-related** (eQTL analysis)
- [x] r-minpack.lm https://github.com/conda-forge/staged-recipes/pull/3883
- [x] r-multitaper https://github.com/conda-forge/staged-recipes/pull/3883
- [x] ~~r-nastiseq~~ **bio-related** (cis-NAT identification from RNA-seq data. Neither CRAN nor Bioconductor package)
- [x] r-nleqslv https://github.com/conda-forge/staged-recipes/pull/3883
- [x] r-nor1mix https://github.com/conda-forge/staged-recipes/pull/3883
- [x] r-nozzle.r1 https://github.com/conda-forge/staged-recipes/pull/3883
- [x] r-openxlsx https://github.com/conda-forge/staged-recipes/pull/3778
- [x] r-outliers https://github.com/conda-forge/staged-recipes/pull/3883
- [x] r-pack https://github.com/conda-forge/staged-recipes/pull/3883
- [x] ~~r-patpro~~ **bio-related** (visualize QIIME output)
- [x] ~~r-peer~~ **bio-related** (batch effects in gene expression)
- [x] r-penalized https://github.com/conda-forge/staged-recipes/pull/3883
- [x] r-pmcmr https://github.com/conda-forge/staged-recipes/pull/3883
- [x] ~~r-proxy~~ (0.4.17 in conda-forge)
- [x] r-prroc https://github.com/conda-forge/staged-recipes/pull/3883
- [x] r-pscl https://github.com/conda-forge/staged-recipes/pull/3778
- [x] ~~r-psych~~ (1.7.3.21 in conda-forge)
- [x] ~~r-qqman~~ **bio-related** (visualizing GWAS data)
- [ ] r-r2html
- [x] r-rann https://github.com/conda-forge/staged-recipes/pull/3877
- [x] ~~r-rappdirs~~ (0.3.1 in conda-forge)
- [x] ~~r-rBLAST~~ **bio-related** (BLAST interface. Not CRAN or Bioc)
- [x] ~~r-rcircos~~ **bio-related** (circos plots. Nominally bio-related; calling it bio-related since the package description explicitly mentions genomic data)
- [x] r-rcppgsl
- [x] r-rcppparallel
- [x] r-rcpptoml
- [x] ~~r-readr~~ (1.1.0 in conda-forge)
- [ ] r-rematch2
- [x] r-rgraphics https://github.com/conda-forge/staged-recipes/pull/3877
- [x] ~~r-robustrankaggreg~~ **bio-related** (aggregating ranked lists; specifically mentions gene lists in the description)
- [x] ~~r-rphylip~~ **bio-related** (interface to PHYLIP package (phylogeny))
- [x] r-rsm
- [x ] r-rtsne https://github.com/conda-forge/staged-recipes/pull/3877
- [x] ~~r-scrm~~ **bio-related** (simulate sequences under neutral model)
- [x] r-shinyace https://github.com/conda-forge/staged-recipes/pull/3877
- [x] r-signal
- [x] r-skellam
- [x] r-som
- [x] r-sparql https://github.com/conda-forge/staged-recipes/pull/3877
- [x] ~~r-speaq~~ **bio-related** (NMR analysis tools)
- [x] r-speedglm https://github.com/conda-forge/staged-recipes/pull/3776
- [x] r-statmod https://github.com/conda-forge/staged-recipes/pull/3776
- [x] r-tfmpvalue  **bio-related** (p-values for PWMs)
- [x] r-tiff https://github.com/conda-forge/staged-recipes/pull/3776
- [x] r-tsne https://github.com/conda-forge/staged-recipes/pull/3776
- [x] r-upsetr https://github.com/conda-forge/staged-recipes/pull/3776
- [x] r-venn https://github.com/conda-forge/staged-recipes/pull/3776
- [x] ~~r-viridis~~ (0.4.0 in conda-forge)
- [x] r-waveslim https://github.com/conda-forge/staged-recipes/pull/3776
- [x] r-wrassp https://github.com/conda-forge/staged-recipes/pull/3776
- [x] r-writexls https://github.com/conda-forge/staged-recipes/pull/3776
- [x] r-xmlrpc [Omegahat package](https://github.com/duncantl/XMLRPC)
- [x] r-xnomial  **bio-related??** (multinomial is general, but all vignette examples are from genetics)

**Comment by jdblischak** [comment]
I marked the remaining packages that are already available on conda-forge. The version string may be slightly different because I converted them to numeric version strings in order to identify the most recent version on conda-forge (using R's `as.numeric_version()`).

I can help with this effort. There are a handful of packages in this list that I was hoping to get into conda-forge. Would it be useful if I submitted a PR with a batch of the singletons? I could start from the bottom of the list to avoid duplicating effort.

**Comment by daler** [comment]
@jdblischak thanks for going through these; starting at the bottom of singletons would be fantastic. You could also work on any subdag that is not already being worked on, since they are independent from each other. That might be a better option for you if you have particular packages you want sooner rather than later.

**Comment by bgruening** [comment]
@daler @jdblischak we were able to merge the first bigger batch yesterday. It's still in the CI queue to get coverted to a feedstock but we have some progress :)
Thanks for helping so much!

**Comment by daler** [comment]
@bgruening great! Can you remind me: once the feedstock is created, it has to be tested again before getting uploaded to the channel?

**Comment by bgruening** [comment]
@daler yes. The feedstock is created by the staged-recipes PR and than a normal run is triggered on the master branch of the feedstock which created the package. I needed to remove 2 recipes from my PR due to build-time outs. So I need to get these in before we can work on your follow-up one.

**Comment by daler** [comment]
Got it. Thanks!

**Comment by daler** [comment]
@bgruening @jdblischak I'm going through all packages to decide on bio-relatedness; please hold off on editing the original issue above so we don't lose any information. I'll report back here as soon as I'm done.

**Comment by daler** [comment]
@bgruening @jdblischak OK, I think I got everything. Should be more straightforward to create conda-forge PRs now.

**Comment by souravsingh** [comment]
@daler I can work on the rest of the R packages.

**Comment by daler** [comment]
@souravsingh that would be fantastic. I'm a bit crunched for time through the next two weeks.

**Comment by bgruening** [comment]
@souravsingh awesome! I have a big one pending and will update the list accordingly. Currently we are blocked by some CI shortage :(

**Comment by bgruening** [comment]
@souravsingh ok I got my next two pushed, if you would like to take over for today :)

**Comment by jdblischak** [comment]
There are currently 6 PRs submitted to conda-forge that have failed CI builds. I've reviewed some of them, but I know many are failing due to missing dependencies. We've merged lots of recipes recently. Should I restart the CI builds on the PRs? Is there any PR in particular that I should investigate? Please let me know if there is anything I can do to help.

**Comment by bgruening** [comment]
@jdblischak I'm on it. Part of the problem is that we have a few circular dependencies which I try to resolve and that I also need to restart several feedstock repos. This just takes a lot of time. Sorry.

**Comment by bgruening** [comment]
I marked a few more as resolved as I could merge a few RPs in the last day. It does not look that bad, we are getting there :)

**Comment by bgruening** [comment]
Ok, I added a bunch more and restarted a few feedstocks to keep the CIs busy over the weekend.

**Comment by bgruening** [comment]
@daler @jdblischak I spend the last days and pushed it over the finish line. I think we have most of the packages, hopefully all. One last PR is open and this will be merged in the next hours.

So **we did it**!
I also removed a lot of packages from bioconda in the bulk branch. I must admit that I lost a little bit track, but I did my best and hopefully this all looks fine for you.

**Comment by jdblischak** [comment]
> So we did it!

Awesome! Great work, @bgruening, on making the final push!

**Comment by daler** [comment]
@bgruening this is so awesome.

I can't imagine the time you've spent over the past weeks babysitting and restarting CI builds across multiple providers and keeping track of which failed due to dependency feedstocks not being complete . . . 

Phew. Amazing work!

**Comment by bgruening** [comment]
As usual there are still things to work on, but I think we archived our goals here and with https://github.com/bioconda/bioconda-recipes/pull/6119 I'm going to close this one.
Thanks all for your help.
